File: C#/Task150.cs
using System;

public class Task150
{
    public static void Main()
    {
        const double a = 7.7; // параметр a, константа

        Console.Write("x = ");
        double x = double.Parse(Console.ReadLine());

        double y = 0.0; // змінна повинна бути ініціалізована

        if (x >= 0 && x <= 5)
        {
            y = x * Math.Pow(a * x * x * x + Math.Sin(x), 1.0 / 3.0);
        }

        if (x < 0)
        {
            y = a * Math.Sin(x) * Math.Sin(x) * Math.Sin(x);
        }

        if (x > 5)
        {
            y = Math.Log(Math.Abs(x * x * x - Math.Exp(x))) / Math.Log(x);
        }

        Console.WriteLine($"y = {y,12:F5}");
    }
}

File: Kotlin/Task150.kt
import kotlin.math.*

fun main() {
    val a = 7.7
    print("x = ")
    val x = readLine()!!.toDouble()

    val y = when {
        x in 0.0..5.0 ->
            x * (a * x * x * x + sin(x)).pow(1.0 / 3)

        x < 0 ->
            a * sin(x).pow(3)

        else ->  // x > 5
            ln(abs(x * x * x - exp(x))) / ln(x)
    }

    println("y = %12.5f".format(y))
}
